demisto_sdk/commands/create_id_set/tests/update_id_set_test.py

```diff
@@ -70,6 +88,67 @@ def test_create_id_set_no_output(self, mocker):
         assert 'Widgets' in id_set.keys()
         assert 'Mappers' in id_set.keys()
 
+    def test_create_id_set_on_specific_pack(self, repo):
+        """
+        Given
+        - two packs with integrations to create an ID set from them
+
+        When
+        - create ID set on pack1
+
+        Then
+        - ensure there is only one integration in the ID set integrations list - integration1
+        - ensure output id_set contains pack_to_create_id_set_on - integration1
+        - ensure output id_set does not contain the pack_to_not_create_id_set_on - integration2
```

Comment:
These lines should not reflect the names of variables in the code.
They should provide a high level explanation of what the test tests, and why

Reply:
done